arrocke/gloss-translation
# docs:Update Dev login credentials

## What has changed


Updated email in contributing.md

## Connected Issues
 

closes #301 
## QA Steps



1. Start `nx serve api`
2.  Start `nx server web`
3. On web page, click on `Log In` menu item
4. Enter user: `platform_admin@example.com` with password: `asdf1234`
5. Click Login In button.
6. You should login successfully with admin privileges.

## Post-Deployment



- [x ] Nothing required
